frontend/public/index.html

```diff
@@ -10,6 +10,11 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css" integrity="sha384-xxzQGERXS00kBmZW/6qxqJPyxW3UR0BPsL4c8ILaIWXva5kFi7TxkIIaMiKtqV1Q" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+
+
```

Comment:
You can remove this -- here are the notes for why.

Two notes:
1. You are not actually using this font -- you can see the differences in the image attached (the edited left side is Montserrat, right side is the default font).
2. You'll want to add the font to the `Theme.tsx` file in `src/components/config` -- then, you'll be able to reference that font with the line `${({ theme }) => theme.font};`. You can add different fonts too, though I'd recommend using the `Roboto` already there as a starter.

<img width="958" alt="Montserrat vs Default Font" src="https://user-images.githubusercontent.com/7987279/107282314-a7f77c00-6a0f-11eb-929b-d370f6da6d05.png">